EosPayload/drivers/I2C_driverIgnore.py

```diff
@@ -0,0 +1,92 @@
+
```

Comment:
what is driverignore?  If this was just something you were using for testing and now aren't using anymore, remove it